lib/store/application.ts

```diff
@@ -7,12 +7,25 @@ import {
   ThunkDispatch,
 } from '@reduxjs/toolkit';
 import { Store } from '.';
-import { Application } from '../../domain/HousingApi';
+import {
+  Application,
+  CreateAuthRequest,
+  CreateAuthResponse,
+  VerifyAuthRequest,
+  VerifyAuthResponse,
+} from '../../domain/HousingApi';
 import { signOut } from './cognitoUser';
 import mainApplicant from './mainApplicant';
 import otherMembers from './otherMembers';
 import { NotifyRequest, NotifyResponse } from '../../domain/govukNotify';
 
+export const importApplication = createAsyncThunk(
+  'applicant/import',
+  async (application: Application) => {
+    return application as Application;
+  }
+);
```

Comment:
This doesn't seem quite right. A thunk would usually perform some kind of async action. This function appears to just return what it's given.